float64[] replan_human_obstacles

---

bool is_replanned